Rubberduck.Parsing/Emitter.cs

```diff
@@ -1,7 +1,9 @@
-﻿using System.Diagnostics;
```

Comment:
I should have removed this class a long time ago... this was a (successful) experiment to see if we could _emit_ a VBA function, run it and retrieve a value. The idea was to use it to return the `TypeName` of each document-type module, as a hack that's now completely moot.